build.proj

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="4.0" DefaultTargets="Test;Chocolatey" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Compile;Test;Chocolatey" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I am trying to separate the compile and test in different build step on Team City in order to activate the code coverage.